drivers/flash/spi_nor.h

```diff
@@ -110,5 +110,6 @@
 /* Test whether offset is aligned to a given number of bits. */
 #define SPI_NOR_IS_ALIGNED(_ofs, _bits) (((_ofs) & BIT_MASK(_bits)) == 0)
 #define SPI_NOR_IS_SECTOR_ALIGNED(_ofs) SPI_NOR_IS_ALIGNED(_ofs, 12)
+#define SPI_NOR_IS_64K_ALIGNED(_ofs) SPI_NOR_IS_ALIGNED(_ofs, 16)
```

Comment:
cool! could you add a 32k as well while we are at it? there's a `SPI_NOR_CMD_BE_32K` command so it may come in handy

Reply:
Added 32k erase in new revision. Thank you